test/backend_test.in.cpp

```diff
@@ -5207,6 +5207,63 @@ NGRAPH_TEST(${BACKEND_NAME}, max_pool_2d_2channel_2image)
               read_vector<float>(result));
 }
 
```

Comment:
Please add a short description to this test that describes the problem it is trying to detect.